<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dl_event_class_manager</class>
 <widget class="QDialog" name="dl_event_class_manager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Event Class Manager</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTableWidget" name="tableWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="pb_new">
         <property name="toolTip">
          <string>Create new Event Class</string>
         </property>
         <property name="text">
          <string>New...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_delete">
         <property name="toolTip">
          <string>Delete selected Event Class</string>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="pb_add">
         <property name="toolTip">
          <string>Add Event Class layers to QGIS</string>
         </property>
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_remove">
         <property name="toolTip">
          <string>Remove layers from QGIS</string>
         </property>
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pb_import">
         <property name="toolTip">
          <string>Import tables as Event Class</string>
         </property>
         <property name="text">
          <string>Import...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_close">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tableWidget</tabstop>
  <tabstop>pb_new</tabstop>
  <tabstop>pb_delete</tabstop>
  <tabstop>pb_add</tabstop>
  <tabstop>pb_remove</tabstop>
  <tabstop>pb_import</tabstop>
  <tabstop>pb_close</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
